src/GraphQLParser.ApiTests/GraphQLParser.approved.txt

```diff
@@ -906,6 +918,7 @@ namespace GraphQLParser.Visitors
         public SDLPrinter() { }
         public SDLPrinter(GraphQLParser.Visitors.SDLPrinterOptions options) { }
         public GraphQLParser.Visitors.SDLPrinterOptions Options { get; }
+        protected virtual System.Threading.Tasks.ValueTask MakeVerticalIndentationBetweenTopLevelDefinitions(GraphQLParser.AST.ASTNode node, TContext context) { }
```

Comment:
New method is needed for descendants that may skip nodes, see `PrintOnlyStartsWithA`. Without it `SDLPrinter` prints large vertical indentation in place of skipped nodes.